reporters/GlobalSummaryReporter.js

```diff
@@ -0,0 +1,237 @@
+    let collateralDeposited = toBN("0");
+    let collateralDepositedDaily = toBN("0");
+    for (let event of depositEvents) {
+      collateralDeposited = collateralDeposited.add(toBN(event.collateralAmount));
+      if (event.blockNumber >= blockNumberOneDayAgo) {
+        collateralDepositedDaily = collateralDepositedDaily.add(toBN(event.collateralAmount));
+      }
+    }
+    for (let event of createEvents) {
+      collateralDeposited = collateralDeposited.add(toBN(event.collateralAmount));
+      if (event.blockNumber >= blockNumberOneDayAgo) {
+        collateralDepositedDaily = collateralDepositedDaily.add(toBN(event.collateralAmount));
+      }
+    }
+    allSponsorStatsTable["collateral deposited"] = {
+      cumulative: this.formatDecimalString(collateralDeposited),
+      "24H": this.formatDecimalString(collateralDepositedDaily),
+      current: this.formatDecimalString(await this.empContract.methods.totalPositionCollateral().call())
+    };
+
+    // - Cumulative collateral withdrawn from contract: Withdraws, Redeems, SettleExpired's, WithdrawLiquidations, RegularFees, FinalFees
+    let collateralWithdrawn = toBN("0");
+    let collateralWithdrawnDaily = toBN("0");
+    for (let event of withdrawEvents) {
+      collateralWithdrawn = collateralWithdrawn.add(toBN(event.collateralAmount));
+      if (event.blockNumber >= blockNumberOneDayAgo) {
+        collateralWithdrawnDaily = collateralWithdrawnDaily.add(toBN(event.collateralAmount));
+      }
+    }
+    for (let event of redeemEvents) {
+      collateralWithdrawn = collateralWithdrawn.add(toBN(event.collateralAmount));
+      if (event.blockNumber >= blockNumberOneDayAgo) {
+        collateralWithdrawnDaily = collateralWithdrawnDaily.add(toBN(event.collateralAmount));
+      }
+    }
+    for (let event of expirySettlementEvents) {
+      collateralWithdrawn = collateralWithdrawn.add(toBN(event.collateralReturned));
+      if (event.blockNumber >= blockNumberOneDayAgo) {
+        collateralWithdrawnDaily = collateralWithdrawnDaily.add(toBN(event.collateralReturned));
+      }
+    }
+    for (let event of liquidationRewardEvents) {
+      collateralWithdrawn = collateralWithdrawn.add(toBN(event.withdrawalAmount));
+      if (event.blockNumber >= blockNumberOneDayAgo) {
+        collateralWithdrawnDaily = collateralWithdrawnDaily.add(toBN(event.withdrawalAmount));
+      }
+    }
+    for (let event of regularFeeEvents) {
+      collateralWithdrawn = collateralWithdrawn.add(toBN(event.regularFee)).add(toBN(event.lateFee));
+      if (event.blockNumber >= blockNumberOneDayAgo) {
+        collateralWithdrawnDaily = collateralWithdrawnDaily.add(toBN(event.regularFee)).add(toBN(event.lateFee));
+      }
+    }
+    for (let event of finalFeeEvents) {
+      collateralWithdrawn = collateralWithdrawn.add(toBN(event.amount));
+      if (event.blockNumber >= blockNumberOneDayAgo) {
+        collateralWithdrawnDaily = collateralWithdrawnDaily.add(toBN(event.amount));
```

Comment:
nit: are we including liquidation and dispute events as "deposits"? If not, won't these final fee events and the expiry settlement and liquidation reward events make it seem like more collateral is being withdrawn than deposited? Meta-comment: It might be easier to just use the events emitted from the collateral currency to track net deposits and withdrawals than the EMP events.

Reply:
We are not including liquidation/dispute events as deposits. So yes those liquidation-related events would contribute to collteral withdrawn, but the follow-on sections to this `GlobalSummaryStatsReport` will show liquidation, dispute, and fee stats

Reply:
>It might be easier to just use the events emitted from the collateral currency to track net deposits and withdrawals than the EMP events.

What do you mean?

Reply:
If we want to track net deposits and withdrawals in a potentially simpler way, we could just use the ERC20 `Transfer` event that's emitted by the collateral currency:

```
event Transfer(address indexed from, address indexed to, uint tokens);
```

For deposits, you could just filter by `to: this.empContract.address`. And for total withdrawals, you could filter by `from: this.empContract.address`. The disadvantage of doing it this way is that you would be including all transfers, meaning you wouldn't be able to differentiate fee payments from liquidation settlements from redemptions. However, if there are only a small number of these types that you care about, you can always just break them out separately.

Just wanted to point it out as an alternative to querying all the different EMP methods and summing them up.

Reply:
I think the advantage of being able to potentially separate out deposits/withdraws from settlements/fees/etc. is worth keeping it as is. I could see @allisonlu22 wanting that broken down in the future.